src/test/duke/customClass/LogicHandlerTest.java

```diff
@@ -0,0 +1,52 @@
+package duke.customClass;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.ArrayList;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LogicHandlerTest {
+
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final PrintStream originalOut = System.out;
+
+    @BeforeEach
+    public void setUpStreams() {
+        System.setOut(new PrintStream(outContent));
+    }
+
+    @Test
+    public void out() {
+        System.out.print("hello");
+        assertEquals("hello", outContent.toString());
+    }
+
+    @Test
+    public void TestTodo() {
+        LogicHandler lh = new LogicHandler();
+        List<Task> tempList = new ArrayList<>();
+        lh.todo("todo homework", tempList);
+        assertEquals("added: [T][✘] homework\nNow you have 1 tasks in the list.\n"
+                , outContent.toString());
```

Comment:
Perhaps the break should be after the comma instead?
I noticed the same issue in several other places too.